jsonnet/telemeter/benchmark/kubernetes.libsonnet

```diff
@@ -9,12 +9,21 @@ local authorizePort = 8083;
 local externalPort = 8080;
 local internalPort = 8081;
 local clusterPort = 8082;
+local benchmarkPort = 8080;
 local tokensFileName = 'tokens.json';
+local timeseriesFileName = 'timeseries.txt';
+local configmapMountPath = '/etc/config';
 
 {
   _config+:: {
     namespace: 'telemeter-benchmark',
 
+    telemeterBenchmark+:: {
+      to: 'https://infogw.api.openshift.com',
```

Comment:
Will this add data to production telemeter?
I was expecting you'd do infogw.api.stage.
(Is there a third one for integration env?)